mm/mm_heap/mm_mallinfo.c

```diff
@@ -92,6 +93,7 @@ int mm_mallinfo(FAR struct mm_heap_s *heap, FAR struct mallinfo *info)
           if ((node->preceding & MM_ALLOC_BIT) != 0)
             {
               DEBUGASSERT(node->size >= SIZEOF_MM_ALLOCNODE);
+              aordblks ++;
```

Comment:
"No space separating unary operators"

https://github.com/apache/incubator-nuttx/blob/master/Documentation/contributing/coding_style.rst#operators

Reply:
> "No space separating unary operators"
> 
> https://github.com/apache/incubator-nuttx/blob/master/Documentation/contributing/coding_style.rst#operators

Done. Thank you.